<?xml version="1.0"?>
<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="markdown" source-language="en-US" target-language="de-de">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-fdd610b" tool-company="Microsoft" />
      <xliffext:olfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">12a5a396293706f7659ad11508a44390fcec1664</xliffext:olfilehash>
      <xliffext:olfilepath xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">System.ServiceModel.CommunicationObjectAbortedException.yml</xliffext:olfilepath>
      <xliffext:oltranslationpriority xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">net47</xliffext:oltranslationpriority>
      <xliffext:oltranslationtype xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">Human Translation</xliffext:oltranslationtype>
      <xliffext:olskeletonhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">ecb4936919355c48248c3c1f9752604844816e90</xliffext:olskeletonhash>
      <xliffext:olxliffhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">b2bb51e39463804be465bba58965d9c6d9253a34</xliffext:olxliffhash>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve">
          <source>The exception that is thrown when the call is to an <ph id="ph1">&lt;xref href="System.ServiceModel.ICommunicationObject"&gt;&lt;/xref&gt;</ph> object that has aborted.</source>
        </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT">
          <source><ph id="ph1">&lt;xref:System.ServiceModel.CommunicationObjectAbortedException&gt;</ph> is a derived class of <ph id="ph2">&lt;xref:System.ServiceModel.CommunicationException?displayProperty=fullName&gt;</ph> and can occur in correct applications.</source>
        </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT">
          <source>Because typically the code that calls the <ph id="ph1">&lt;xref:System.ServiceModel.ICommunicationObject.Abort%2A?displayProperty=fullName&gt;</ph> method reports the error condition that caused the call to <ph id="ph2">&lt;xref:System.ServiceModel.ICommunicationObject.Abort%2A?displayProperty=fullName&gt;</ph>, this exception indicates that the call failed but the most useful error information is elsewhere.</source>
        </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <ph id="ph1">&lt;xref href="System.ServiceModel.CommunicationObjectAbortedException"&gt;&lt;/xref&gt;</ph> class.</source>
        </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <ph id="ph1">&lt;xref href="System.ServiceModel.CommunicationObjectAbortedException"&gt;&lt;/xref&gt;</ph> class with the specified message.</source>
        </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT">
          <source>Creates a new <ph id="ph1">&lt;xref:System.ServiceModel.CommunicationObjectAbortedException&gt;</ph> using the specified <ph id="ph2">`message`</ph>.</source>
        </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve">
          <source>The message carried in the exception.</source>
        </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <ph id="ph1">&lt;xref href="System.ServiceModel.CommunicationObjectAbortedException"&gt;&lt;/xref&gt;</ph> class used to deserialize data into a <ph id="ph2">&lt;xref href="System.ServiceModel.CommunicationObjectAbortedException"&gt;&lt;/xref&gt;</ph> object.</source>
        </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT">
          <source>This constructor is called during deserialization to restore the exception object transmitted over a stream.</source>
        </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve">
          <source>The <ph id="ph1">&lt;xref href="System.Runtime.Serialization.SerializationInfo"&gt;&lt;/xref&gt;</ph> that holds the serialized object data about the exception being thrown.</source>
        </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve">
          <source>The <ph id="ph1">&lt;xref href="System.Runtime.Serialization.StreamingContext"&gt;&lt;/xref&gt;</ph> that contains contextual information about the source or destination.</source>
        </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <ph id="ph1">&lt;xref href="System.ServiceModel.CommunicationObjectAbortedException"&gt;&lt;/xref&gt;</ph> class using the specified message and inner exception.</source>
        </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT">
          <source>Creates a new <ph id="ph1">&lt;xref:System.ServiceModel.CommunicationObjectAbortedException&gt;</ph> using the specified <ph id="ph2">`message`</ph> and <ph id="ph3">`innerException`</ph>.</source>
        </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve">
          <source>The exception message.</source>
        </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve">
          <source>The inner exception.</source>
        </trans-unit>
      </group>
    </body>
  </file>
</xliff>